examples/log-extension/src/__tests__/acceptance/log.extension.acceptance.ts

```diff
@@ -3,44 +3,43 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+import {inject} from '@loopback/context';
+import {get, param} from '@loopback/openapi-v3';
 import {
-  RestApplication,
-  SequenceHandler,
-  RestBindings,
   FindRoute,
-  ParseParams,
   InvokeMethod,
-  Send,
+  ParseParams,
   Reject,
   RequestContext,
+  RestApplication,
+  RestBindings,
+  Send,
+  SequenceHandler,
 } from '@loopback/rest';
-import {get, param} from '@loopback/openapi-v3';
 import {
-  LogMixin,
-  LOG_LEVEL,
-  log,
-  EXAMPLE_LOG_BINDINGS,
-  LogFn,
-  HighResTime,
-} from '../..';
-import {
-  sinon,
-  SinonSpy,
   Client,
   createClientForHandler,
   expect,
+  sinon,
+  SinonSpy,
 } from '@loopback/testlab';
-import {inject} from '@loopback/context';
 import chalk from 'chalk';
+import {
+  EXAMPLE_LOG_BINDINGS,
+  HighResTime,
+  log,
+  LogFn,
+  LogMixin,
+  LOG_LEVEL,
+} from '../..';
+import {logToMemory, resetLogs} from '../in-memory-logger';
+import {createLogSpy, restoreLogSpy} from '../log-spy';
 
 const SequenceActions = RestBindings.SequenceActions;
 
-import {createLogSpy, restoreLogSpy} from '../log-spy';
-import {logToMemory, resetLogs} from '../in-memory-logger';
-
 describe('log extension acceptance test', () => {
   let app: LogApp;
-  let spy: SinonSpy;
+  let spy: SinonSpy<[(string | undefined)?], void>;
```

Comment:
There was an error showing up while running tests. It needed explicit definition of type for the generic param. So I added it. Rest of the file changes are just reordering of imports done by vscode extension TSHero. I hope that's fine.

Reply:
I think these new errors are reported by the newer version of TypeScript compiler that we recently upgraded to. Can you please rebase your patch on top of the latest master? I think the log-extension example should be already fixed by now.

/cc @raymondfeng

Reply:
I already did. Let me run the tests again and check.

Reply:
works. I removed my changes from here.